peer.go

```diff
@@ -1076,6 +1092,13 @@ out:
 			delay := (time.Now().UnixNano() - pingSendTime) / 1000
 			atomic.StoreInt64(&p.pingTime, delay)
 
+			// Notify pingHandler we've received a pong message.
```

Comment:
can we just forward ping/pong messages to the pinghandler, and move all the associated logic there?

Reply:
i'm okay with it here, as the ping time calculation will be more accurate